src/app/shared/services/nav-types.ts

```diff
@@ -39,3 +39,7 @@ export function itemFromDetailParams(params: ParamMap): NavItem|undefined {
     itemPath: pathAsString === null || pathAsString.length === 0 ? [] : pathAsString.split(','),
   };
 }
+
+export function pathGiven(params: ParamMap): boolean {
```

Comment:
```suggestion
export function isPathGiven(params: ParamMap): boolean {
```
That would require some other changes, but I feel like it would make what the fonction does clearer

Reply:
👍  done